tools/serverpod_cli/lib/src/analyzer/dart/endpoints_analyzer.dart

```diff
@@ -40,11 +40,10 @@ class EndpointsAnalyzer {
   Future<bool> updateFileContexts(Set<String> filePaths) async {
     await _refreshContextForFiles(filePaths);
 
-    var oldDefinitions = _endpointDefinitions;
+    var oldDefinitionsLength = _endpointDefinitions.length;
```

Comment:
This naive check is enough since we return true in the other case (when an endpoint file is changed).